HenHen.Core/Worlds/DateTime.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Affectionate Dove <contact@affectionatedove.com>.
+// Licensed under the Affectionate Dove Limited Code Viewing License.
+// See the LICENSE file in the repository root for full license text.
+
+namespace HenHen.Core.Worlds
+{
+    public struct DateTime
```

Comment:
Should also have properties indicating e.g. what minute it is in the current hour.
If `Minutes` returns 76, `Minute` should return 16.